torch/testing/_internal/inductor_utils.py

```diff
@@ -79,6 +85,18 @@ def decorate_fn(fn):
 
     return decorate_fn
 
+def skip_windows_ci(name: str, file: str) -> None:
+    if IS_WINDOWS and IS_CI:
+        module = os.path.basename(file).strip(".py")
+        sys.stderr.write(
+            f"Windows CI does not have necessary dependencies for {module} tests yet\n"
+        )
+        if name == "__main__":
+            sys.exit(0)
+        raise unittest.SkipTest("requires sympy/functorch/filelock")
+
+requires_gpu = functools.partial(unittest.skipIf, not HAS_GPU, "requires gpu")
```

Comment:
Another quality of life refactor

Reply:
Can you rewrite the other test files to use this?